qt/aqt/profiles.py

```diff
@@ -708,3 +708,15 @@ def set_network_timeout(self, timeout_secs: int) -> None:
 
     def network_timeout(self) -> int:
         return self.profile.get("networkTimeout") or 60
+
+    def set_ankihub_token(self, val: str | None) -> None:
+        self.profile["ankiHubToken"] = val
```

Comment:
I suggest adding a prefix to the third-party keys. The name spaces will be a mess when other services are added